sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -372,6 +373,19 @@ public EditEventSteps(
           webDriverHelpers.clickOnWebElementBySelector(EditEventPage.CREATE_EVENT_HANDOUT_BUTTON);
           webDriverHelpers.clickOnWebElementBySelector(EditEventPage.CANCEL_EVENT_HANDOUT_BUTTON);
         });
+    And(
+        "I check that number of displayed Event Participants is {int}",
+        (Integer number) -> {
+          webDriverHelpers.clickOnWebElementBySelector(EditEventPage.EVENT_PARTICIPANTS_TAB);
```

Comment:
move out this logic to be reused in other places, and keep in this method only the check

Reply:
Can you give me explanation what do you mean exactly? This logic is widely used across project and I don't know which logic to move and what leave to check :D